Group_Constitution

```diff
@@ -0,0 +1,127 @@
+Constitution of “Mozilla Science Study Group- University of Toronto”
+
+1.	Name
+
+The official name of this recognized campus group is “     ”
+
+The acronym or abbreviation of this group is:      
+
+2.	Purpose and Objectives
+
+The purpose, objectives, mission and/or mandate of organization is outlined here:
+     
+
+3.	Membership
+
+Membership to the group is open to all the University of Toronto members (students, staff, faculty and alumni). 
+
+U of T members are permitted to run, nominate, and vote in elections and constitutional amendments. The group is open to non-U of T members. However, these members do not hold the aforementioned rights. Members must register with a designated executive by submitting their full name and a valid email address.
+
+The membership fee will be $      per year.
+
+Any member of the club may apply for a refund within one (1) month of becoming a member. No refunds are permitted 30 days after the election periods.
+
+For recognition by the University of Toronto Students’ Union (UTSU), the group must maintain a minimum of 30 members, a total of 51% of membership are UTSU members. The group must also maintain recognition from the Office of Student Life. These requirements are subject to change and should be checked with UTSU annually to ensure qualifications are met. 
+
+4.	Executive List and Duties
+
+The executive committee shall be comprised of three (3) elected officers. These include a President, Secretary, and Treasurer.
+
+The President shall:
+•	Oversee the operations, management and success of the group
+•	Be the spokesperson for the group
+•	Hold signing officer authority along with the Treasurer for financial purposes
+•	Preside over board meetings as well as general meetings
+•	Ensure transition of office to the future Executives
+Additional responsibilities may include: 
+     
+
+
+The Vice-President shall:
+•	Assume duties of the President in his/her absence
+•	Oversee the various committees
+•	Ensure that all the activities of the club meet regulations and policies of the University of Toronto
+•	Coordinate organizational recruitment efforts
+Additional responsibilities may include: 
+     
+
+
+The Secretary shall: 
+•	Make a list of all registered members 
+•	Maintain the web sites and member contact list
+•	Record notes and motions for meetings
+•	Notify all members of meetings
+•	Handle official correspondence of the organization
+Additional responsibilities may include: 
+     
+
+
+
+The Treasurer shall:
+•	Record all financial transactions of the group
+•	Hold signing officer authority along with the President for financial purposes
+•	Maintain a budget of income and expenses along with receipts
```

Comment:
@lwjohnst86: I decided to respond to one of your comments instead of all of them: I like them all! I'll add them in to the next draft.